Source/WebCore/Makefile

```diff
@@ -4,5 +4,3 @@ PDFJS_SRCROOT = $(SRCROOT)$(PATH_FROM_ROOT)/../ThirdParty/pdfjs
 $(PDFJS_SRCROOT):
 	mkdir -p $@
 	cp -R ../ThirdParty/pdfjs/ $(PDFJS_SRCROOT)
-
-installsrc: $(PDFJS_SRCROOT)
```

Comment:
I think you can delete the above $(PDFJS_SRCROOT) rule too.

Reply:
Done. To be entirely honest, I don't know makefile syntax enough to understand what's happening in these few lines.